data/input_2022/TD/node-wot/node-wot.csv

```diff
@@ -1,109 +1,109 @@
-tm-compose-submodel,pass,If it is desired to provide information that a Thing Model consists of one or more (sub-)Thing Models
-tm-context-requirement,pass,Additionally, in order to identify it as a JSON-LD document, Thing Model definitions MUST use the keyword @context at top level with same rules as a Thing Description.
-tm-overwrite-interaction,pass,A Thing Model SHOULD NOT overwrite the JSON names defined within the properties, actions, and/or events Map of the extended Thing Model.
-tm-overwrite-types,pass,Definitions SHOULD NOT be overwritten in such a way that possible instance values are no longer valid compared to the origin extended definitions.
-tm-placeholder-replacement,pass,The placeholder labeling MUST be substituted with a concrete value (e.g., as JSON number, JSON string, JSON object, etc) when TD instance is created from the Thing Model.
-tm-placeholder-retyping,pass,If a non string-based value of a JSON name-value pair has a placeholder, the value MUST be (temporarily) typed as string.
-tm-placeholder-usecase,pass,In a case where TD instance terms, but not their values, are known in advance, the placeholder labeling MAY be used in a Thing Model.
-tm-protocol-security-restriction,pass,A Thing Model MAY NOT contain instance specific Protocol Binding and security information such as endpoint addresses.
-tm-ref-recursive-extensions,pass,Recursive extensions leading to an infinite loop MUST NOT be defined.
-tm-tmRef-overwrite-possibility,null,If the intention is to override an existing JSON name-value pair definition from tm:ref, the same JSON name MUST be used at the same level of the tm:ref declaration that provides a new value.
-tm-tmRef-overwrite-process,null,The process to overwrite MUST follow the JSON Merge Patch algorithm as defined in [RFC7396] where the content of the referenced definition is patched with the new provided JSON name-value pairs.
-tm-tmRef-overwrite-semantic-meaning,null,Similar to tm:extends and to keep the semantic meaning, definitions SHOULD NOT be overwritten in such a way that possible instance values are no longer valid compared to the origin referenced definition.
-tm-tmRef-usecase,pass,For importing pieces of definitions of one or more existing Thing Models, the tm:ref term is introduced that provides the location of an existing (sub-)definition that SHOULD be reused.
-tm-tmRef2,null,Every time tm:ref is used, the referenced pre-definition and its dependencies (e.g., by context extension) MUST be assumed at the new defined definition.
-tm-tmRequired-resolver,TODO,The JSON Pointers of tm:required MUST resolve to an entire interaction affordance Map definition.
-tm-versioning,null,When the Thing Model definitions change over time, this SHOULD be reflected in the version container.
+"ID", "Status", "Comment", "Description"
```

Comment:
Please use Assertion instead of Description

Reply:
I see. I just just picked another sample that I think came from you ;-)
https://github.com/w3c/wot-testing/blob/main/data/input_2022/TD/editdor/editdor.csv